src/PhaseState.jl

```diff
@@ -6,32 +6,32 @@ using ReverseDiff
 using RecursiveArrayTools
 using ModelingToolkit
 
-@inline function calcgibbs(ph::U,T::W) where {U<:IdealPhase,W<:Real}
```

Comment:
I know I'm going to have to rebase the Marcus kinetics PR on this one and I know it's going to be very unpleasant if every line in PhaseState.jl has been changed. Can we not format this file here?